### ISSUE TITLE
[Discussion]:Should ballistic shield deflect damage without ever harming pawn?

### ISSUE THREAD

**Issue by masakitenchi** [CombatExtended-Continued/CombatExtended#2412]
This actually caused my concern from somebody on discord
https://discord.com/channels/278818534069501953/324222101550661663/1089562618353696848
I replied him with what I had researched at that moment, thought that he was complaining that something doesn't exist. But when I was testing, what he had said was really counter-intuitive:
My test:

- 762x51mm sabot (25mm sharp, 86.28MPa)
- Legendary ballistic shield (28mm RHA, 42MPa)

With this combination, when the bullet gets shielded, the victim won't receive any damage.
But with similar armor (Legd Recon Armor has ~28mm RHA & ~59.50MPa armor), the victim will always receive some amount of bruise damage.
Considering that pawns are holding their shield with their right arm, it's not reasonable that they fully ignore the impact the bullet has, doesn't it? The impact should at least transmit some portion of force to their arms

TLDR
My opinion is that shield should also check its blunt armor when deflecting sharp attack (like apparel does), especially when blocking melee attacks. For ranged attack, there could be a chance that it "perfectly" deflects that, depending on the pawn's melee skill.

**Comment by masakitenchi** [comment]
Close this as implemented in #2413
